docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,145 @@
+.NET Core
+=========
+
+.NET Core is a general purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform, supporting Windows, OS X and Linux and can be used in device, cloud and embedded/IoT scenarios. The supported Operating Systems (OS), CPUs and application scenarios will grow over time, provided by Microsoft, other companies and individuals.
+
+The following characteritics best define .NET Core:
+
+- **Flexible deployment:** app-local or side-by-side user- or machine-wide installation.
+- **Cross-platform:** Runs on Windows, OS X and Linux; Can be ported to other OSes.
+- **Compatibility:** .NET Core is compatible with .NET Framework and Xamarin platforms.
```

Comment:
Is it worth emphasizing that .NET Core, .NET Framework, and Xamarin all share .NET Standard as a base?